package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "devDependencies": {
+    "@percy/cli": "^1.0.0-beta.46"
```

Comment:
Is this a real dev dep? If so, what is it used for in development? Could it be replaced with `@percy/core` like in the JS SDKs to avoid all the CLI specific stuff? Or is the CLI specific stuff what we want here?